scripts/remote_deployment/deploy.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+
+# IPC Quick Start Script
+# See also https://github.com/consensus-shipyard/ipc/blob/main/docs/ipc/quickstart-calibration.md
+
+# Known issues:
+# 1. Need to previously manual enable sudo without password on the host
+# 2. You may need to rerun the script after docker installation for the first time
+
+# Exit on any error
+set -e
```

Comment:
set -euo pipefail

or

set -eu
set -o pipefail
(both do the same. the pipefail is a bash specific rule that ensures
false|true throws an error - otherwise it only uses the exit code of the last process in the pipeline)
instead of only

set -e
 is what we'd like to add on our bash scripts to ensure they actually exit when there is an error